lightning/src/ln/functional_test_utils.rs

```diff
@@ -779,6 +779,26 @@ pub fn get_revoke_commit_msgs<CM: AChannelManager, H: NodeHolder<CM=CM>>(node: &
 	})
 }
 
+/// Gets a `revoke_and_ack` and `UpdateHTLCs` (i.e. after we get a responding `commitment_signed`
+/// while we have updates in the holding cell).
```

Comment:
Small nit: Maybe we can update the comment to match the event order:

```suggestion
/// Gets a `UpdateHTLCs` and `revoke_and_ack` (i.e. after we get a responding `commitment_signed`
/// while we have updates in the holding cell).
```